op-bootnode/bootnode/entrypoint.go

```diff
@@ -60,6 +60,10 @@ func Main(cliCtx *cli.Context) error {
 	if err != nil {
 		return fmt.Errorf("failed to load p2p config: %w", err)
 	}
+	if p2pConfig.EnableReqRespSync {
```

Comment:
FYI we will always hit this case unless the bootnode operator disables P2P req/resp manually b/c it is on by default.